ui/components/app/edit-gas-fee-popover/edit-gas-tooltip/index.scss

```diff
@@ -1,15 +1,12 @@
 .edit-gas-tooltip {
```

Comment:
Thanks for cleaning the code! I double-checked to see that this specific class, `.edit-gas-tooltip`, is not declared in the codebase